migrations/1716653153304-projectActive.ts

```diff
@@ -0,0 +1,14 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class ProjectActive1716653153304 implements MigrationInterface {
+    name = 'Migrations1716653153304'
```

Comment:
Je sais pas trop pourquoi mais j'avais une erreur "Duplicate migration", en faisant correspondre ce name au nom de la classe comme ailleurs c'est passé

```suggestion
    name = 'ProjectActive1716653153304'
```